biodatacatalyst-ui/Dockerfile

```diff
@@ -29,5 +29,11 @@ COPY src/main/webapp/picsureui /usr/local/apache2/htdocs/picsureui/
 COPY src/main/webapp/psamaui /usr/local/apache2/htdocs/picsureui/psamaui/
 COPY src/main/resources/favicon.ico /usr/local/apache2/htdocs/
 
+COPY --from=pic-sure-ui /usr/local/apache2/htdocs/picsureui/index.html /usr/local/apache2/htdocs/picsureui/index_20210217.html
+COPY --from=pic-sure-ui /usr/local/apache2/htdocs/picsureui/main.js /usr/local/apache2/htdocs/picsureui/main_20210217.js
+
+RUN sed -i 's/\/picsureui\/main/\/picsureui\/main_20210217/' /usr/local/apache2/htdocs/picsureui/index_20210217.html
+RUN sed -i 's/urlArgs: ""/urlArgs: "20210217"/' /usr/local/apache2/htdocs/picsureui/main_20210217.js
```

Comment:
I think you need to provide an argument and value here.  From the requireJS documentation: 

urlArgs: Extra query string arguments appended to URLs that RequireJS uses to fetch resources. Most useful to cache bust when the browser or server is not configured correctly. Example cache bust setting for urlArgs:

urlArgs: "bust=" +  (new Date()).getTime()


... and if that works, do we really need to rename the index files?